x/tokenomics/keeper/settle_session_accounting_test.go

```diff
@@ -47,8 +47,8 @@ func TestSettleSessionAccounting_HandleAppGoingIntoDebt(t *testing.T) {
 		SessionHeader: &sessiontypes.SessionHeader{
 			ApplicationAddress: app.Address,
 			Service: &sharedtypes.Service{
-				Id:   "svc1",
-				Name: "svcName1",
+				Id:   "0021",
```

Comment:
Let's revert this for now.

We haven't made a decision on service IDs in shannon, but there's an opportunity to make them semantic (e.g. `ETH`), JWTs, structures, etc.... 

We'll discuss this in the future, but let's not port over legacy yet.

Leaving one comment but ditto elsewhere.